Resources/templates/CommonAdmin/ListTemplate/ResultsBuilderTemplate.php.twig

```diff
@@ -22,6 +22,20 @@
         <div class="box box-primary">
             <div class="box-header list-header list-nb-results">
                 {{- block('list_nbresults') -}}
+{% if builder.filtersMode == 'modal' %}
+    {{ echo_set('class', '') }}
+    {{ echo_if("app.session.get('\\\\" ~ bundle_name ~ '\\\\' ~ builder.ModelClass ~ "List\\\\Filters',{}) is empty") }}
```

Comment:
What about using something coming from the Controller whereas a test on the session here?

Just a question, don't know what's the better.